examples/cpp-capture.cpp

```diff
@@ -38,65 +44,73 @@ int main(int argc, char * argv[]) try
-            {
-                int value = !dev->get_option(rs::option::r200_emitter_enabled);
-                std::cout << "Setting emitter to " << value << std::endl;
-                dev->set_option(rs::option::r200_emitter_enabled, value);
-            }
-            break;
-        case GLFW_KEY_A:
-            if(dev->supports_option(rs::option::r200_lr_auto_exposure_enabled))
             {
-                int value = !dev->get_option(rs::option::r200_lr_auto_exposure_enabled);
-                std::cout << "Setting auto exposure to " << value << std::endl;
-                dev->set_option(rs::option::r200_lr_auto_exposure_enabled, value);
+            case GLFW_KEY_R: color_rectification_enabled = !color_rectification_enabled; break;
+            case GLFW_KEY_C: align_color_to_depth        = !align_color_to_depth; break;
+            case GLFW_KEY_D: align_depth_to_color        = !align_depth_to_color; break;
+            case GLFW_KEY_E:
+                if(dev->supports_option(rs::option::r200_emitter_enabled))
+                {
+                    int value = !dev->get_option(rs::option::r200_emitter_enabled);
+                    std::cout << "Setting emitter to " << value << std::endl;
+                    dev->set_option(rs::option::r200_emitter_enabled, value);
+                }
+                break;
+            case GLFW_KEY_A:
+                if(dev->supports_option(rs::option::r200_lr_auto_exposure_enabled))
+                {
+                    int value = !dev->get_option(rs::option::r200_lr_auto_exposure_enabled);
+                    std::cout << "Setting auto exposure to " << value << std::endl;
+                    dev->set_option(rs::option::r200_lr_auto_exposure_enabled, value);
+                }
+                break;
             }
-            break;
-        }
     });
     glfwMakeContextCurrent(win);
 
-    while (!glfwWindowShouldClose(win))
+    while(!glfwWindowShouldClose(win))
     {
         // Wait for new images
         glfwPollEvents();
         dev.wait_for_frames();
 
         // Clear the framebuffer
-        int w,h;
+        int w, h;
         glfwGetFramebufferSize(win, &w, &h);
         glViewport(0, 0, w, h);
         glClear(GL_COLOR_BUFFER_BIT);
 
-        // Draw the images        
+        // Draw the images
         glPushMatrix();
         glfwGetWindowSize(win, &w, &h);
         glOrtho(0, w, h, 0, -1, +1);
-        buffers[0].show(dev, align_color_to_depth ? rs::stream::color_aligned_to_depth : (color_rectification_enabled ? rs::stream::rectified_color : rs::stream::color), 0, 0, w/2, h/2);
-        buffers[1].show(dev, align_depth_to_color ? (color_rectification_enabled ? rs::stream::depth_aligned_to_rectified_color : rs::stream::depth_aligned_to_color) : rs::stream::depth, w/2, 0, w-w/2, h/2);
-        buffers[2].show(dev, rs::stream::infrared, 0, h/2, w/2, h-h/2);
-        buffers[3].show(dev, rs::stream::infrared2, w/2, h/2, w-w/2, h-h/2);
+        buffers[0].show(dev,
```

Comment:
@ds-hwang The line break rule greatly reduces readability compared to what's currently used here